clio-dataaccess-model/src/main/scala/org/broadinstitute/clio/server/dataaccess/elasticsearch/Elastic4sAutoDerivation.scala

```diff
@@ -4,7 +4,7 @@ import io.circe.Json
 import org.broadinstitute.clio.util.json.ModelAutoDerivation
 
 /** Automatic JSON encoding and decoding for the Elasticsearch DAO. */
-object Elastic4sAutoDerivation extends ModelAutoDerivation {
+trait Elastic4sAutoDerivation extends ModelAutoDerivation {
```

Comment:
The name of this class seems bad to me. I guess it was chosen to match `com.sksamuel.elastic4s.` but that package isn't used here. Seems like writing out `ElasticsearchAutoDerivation` would be better.

Reply:
I think it's named this way because it's providing the `implicit printer` argument to [this method](https://github.com/sksamuel/elastic4s/blob/master/elastic4s-circe/src/main/scala/com/sksamuel/elastic4s/circe/package.scala#L41) in elastic4s-circe. I'll update the scaladoc to make it more clear what the trait provides.